linkerd/docs/routers.md

```diff
@@ -269,5 +273,14 @@ Key | Default Value | Description
 --- | ------------- | -----------
 minSize | `0` | The minimum number of connections to maintain to each host.
 maxSize | Int.MaxValue | The maximum number of connections to maintain to each host.
-idleTimeMs | forever | The amount of idle time for which a connection is cached in milliseconds.
+idleTimeMs | forever | The amount of idle time for which a connection is cached in milliseconds. Only applied to connections that number greater than minSize, but fewer than maxSize.
```

Comment:
I am trying to understand this comment. How do `HostConnectionPool`'s idleTimeMs differ from `ClientSession`'s idleTimeMs? Do they effectively do the same thing? If they do, then we may no longer need to have this setting in `HostConnectionPool`.

Reply:
They are similar but slightly different.  This connection timeout is only applied to connections between the minSize and maxSize whereas the clientSessions timeouts apply to all connections.  The connectionPool timeout can be thought of as the amount of time "extra" connections are kept around.

Reply:
Although, in the case where minSize=0 (which is the default) then these two timeouts do exactly the same thing.

Reply:
Ahh I see, thanks for the clarification!